### PR TITLE
Make upload work

### PR DESCRIPTION
**Pull request by individual-it**
1. fst used to stop the staetup process with: 
    ```
    fst           | Setting /tmp to mode 1777
    fst           | Unmounting any existing mounts for file systems
    fst           | ls: cannot access /hostdev/mapper/: No such file or directory
    fst           | ls: cannot access /dev/mapper/luks*: No such file or directory
    fst           | ls: cannot access /dev/mapper/*: No such file or directory
    fst           | trying to unlock luks encrypted disk with no passphrase, what are you doing
    ```
    so set some passphrase so that the script runs, even it does not decrypt anything

2. the default space was created but not enabled. see http://eos-docs.web.cern.ch/eos-docs/quickstart/admin/configure.html#configure-mgm

test: 
```
$ ME=artur-neumann
$ SERVER_NAME=$ME-upload-test
$ hcloud server create --type cx21 --image ubuntu-20.04 --ssh-key $ME --name $SERVER_NAME --label owner=$ME --label for=test --label from=eos-compose
$ IPADDR=$(hcloud server ip $SERVER_NAME)
$ ssh -t root@$IPADDR apt-get update -y
$ ssh -t root@$IPADDR apt-get install -y git screen docker.io docker-compose
$ ssh -t root@$IPADDR git clone https://github.com/owncloud-docker/compose-playground.git
$ ssh -t root@$IPADDR "cd compose-playground/examples/eos-compose && git checkout make-upload-work && ./build"
$ ssh -t root@$IPADDR "cd compose-playground/examples/eos-compose && echo OCIS_DOMAIN=$IPADDR > .env && docker-compose up -d"
$ curl -k -u einstein:relativity -X PUT https://$IPADDR:9200/remote.php/dav/files/e/einstein/file.txt -d "123" -v
$ curl -k -u einstein:relativity -X PROPFIND https://$IPADDR:9200/remote.php/dav/files/e/einstein | xmllint --format -
$ curl -k -u einstein:relativity  https://$IPADDR:9200/remote.php/dav/files/e/einstein/file.txt

### CODE REVIEW

File: examples/eos-compose/eos-fst/setup [hunk 1]
```diff
@@ -7,4 +7,5 @@ for i in {1..4}; do
   mkdir -p /disks/eosfs${i}
   chown daemon:daemon /disks/eosfs${i}
   eos -r 0 0 -b fs add eosfs${i} fst.testnet:1095 /disks/eosfs${i} default rw
+  eos -r 0 0 space set default on
```
Comment (felixboehm) [inline examples/eos-compose/eos-fst/setup:10]: what is the reason for this?
And why in fst instead of mgm node?

**Comment by felixboehm** [comment]
I'd like to understand the eos config you added, then I would add it to the images.
Images have moved now to https://github.com/owncloud-docker/eos-stack

File: examples/eos-compose/eos-fst/setup (hunk 1 above)
Comment (individual-it) [reply examples/eos-compose/eos-fst/setup:10]: it looks like it does not make any difference where that command is run to enable the space, but its more about when its run

in eosmgm/setup the same command is executed and if I add a `sleep 30` before it, the spaces are enabled

to check do
`eos health -a `

```
┌────────────┬────────────┬────────────┬────────┐
│group       │offline used│ online free│  status│
└────────────┴────────────┴────────────┴────────┘
 default.0       781.49 GB          0 B     full 
```

vs.

```
┌────────────┬────────────┬────────────┬────────┐
│group       │offline used│ online free│  status│
└────────────┴────────────┴────────────┴────────┘
 default.0             0 B    213.10 GB       ok 
```

so I guess the enable command has to be run after the `fs add eosfs ...`

Comment (jnweiger) [reply examples/eos-compose/eos-fst/setup:10]: Yeah! That is the magic incantation! Same for me in https://github.com/owncloud/ocis/issues/396 , the storage is 'offline' by default, and needs to be switched online with `eos -r 0 0 space set default on`

**Pull request closed without merging**